DockerfileJupyter

```diff
@@ -0,0 +1,42 @@
+#
+#	LEAN Jupyter Docker Container 20170802
+#	
+
+# Use base system for cleaning up wayward processes
+FROM quantconnect/lean:foundation
+
+MAINTAINER QuantConnect <contact@quantconnect.com>
+
+# Install Tini
+RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini && \
+    echo "1361527f39190a7338a0b434bd8c88ff7233ce7b9a4876f3315c22fce7eca1b0 *tini" | sha256sum -c - && \
+    mv tini /usr/local/bin/tini && \
+    chmod +x /usr/local/bin/tini
+
+#Install Jupyter
+RUN pip install jupyter
+	
+# Install Python.NET
+RUN apt-get update && apt-get install -y libglib2.0-dev \
```

Comment:
In the foundation, we have a pythonnet installation that only works in Lean. For instance, if you open python and type "import clr", it says the package is not installed.
We could change the foundation to include this installation method that is more general.